distributions/aws/examples/storage/efs/README.md

```diff
@@ -146,19 +146,29 @@ data_dir = pathlib.Path(data_dir)
 In the `training-sample` directory, we have provided a sample training script and Dockerfile which you can use as follows to build a docker image. Be sure to point the `$IMAGE_URI` to your registry and specify an appropriate tag - 
 ```
 export IMAGE_URI=<dockerimage:tag>
-cd storage-efs/training-sample
+cd training-sample
+
+# You will need to login to ECR for the following steps
```

Comment:
nit: You will need to login to your docker registry for the following steps

Reply:
no, has to be ECR public registry atleast for the base docker image. Plus dockerhub if they want to push there.